env0/data_agent_values_test.go

```diff
@@ -63,5 +63,4 @@ func TestAgentValues(t *testing.T) {
 			},
 		)
 	})
-
```

Comment:
ignore all these changes... there are thousands of required changes required to successfully add a new linter, so I slowly fix these with each PR I make.